curriculum/js1/4.js

```diff
@@ -1,15 +1,22 @@
 /**
  * Print String - takes in a number n and string,
  *     return the string repeated n number of times.
- * @param {number} a
+ a* : @param {number} a
  * @param {string} b
- * @returns {string}
+ *a @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
-};
+
+const solution  = (times, word) => {
```

Comment:
`const solution = (times, word, result='') => {`

Pass down the result into the next function.